### PR TITLE
fix #293 min tics use their own line definition

### LINKED ISSUE #293: templates xmintic line ignored in favor of xtic
```python
import MV2
import numpy
import vcs
x=vcs.init(bg=True)
angle = numpy.arange(0, 360, 10) # in degrees
s = numpy.sin(angle/180.*numpy.pi) # sin takes radians as input
line = vcs.create1d()
line.linecolor = "red"  # Red color obviously
line.linewidth = 2.  # A bit thicker
line.marker = "circle"  #
line.markersize = 10  # A bit bigger
s = MV2.array(s)
s.id = "sine"
xaxis = s.getAxis(0)
xaxis.id = "angle"
xaxis.units="radians"
# let's make them radians as well
xaxis[:] = xaxis[:]/18.*numpy.pi
# That means we need to match or symbols to new values
angles = {0:'0',numpy.pi/2.:r'$\pi/2$',numpy.pi:r'$\pi$', 3*numpy.pi/2.:r'$3\pi/2$',2*numpy.pi:r'$2\pi$'}
line.xticlabels1 = angles
thick = vcs.createline()  # for thick ticks
thick.width = 1.5  # A bit thicker
thick.color = ["grey"]
dots = vcs.createline()  # for thin/sub ticks
dots.type = ["dot"]
dots.color = ["blue"]
tmpl = vcs.createtemplate()
tmpl.xtic1.y2 = tmpl.data.y2  # Ticks extends all the way accross the data section
tmpl.xtic1.line = thick
tmpl.xmintic1.y2 = tmpl.data.y2  # Ticks extends all the way accross the data section
tmpl.xmintic1.priority = 1
tmpl.xmintic1.line = dots
tmpl.ytic1.x2 = tmpl.data.x2  # Ticks extends all the way accross the data section
tmpl.ytic1.line = thick
tmpl.ymintic1.x2 = tmpl.data.x2  # Ticks extends all the way accross the data section
tmpl.ymintic1.priority = 1
tmpl.ymintic1.line = dots
line.xmtics1 = {numpy.pi/4.:"",3*numpy.pi/4:"",5*numpy.pi/4:"",7*numpy.pi/4:""}
x.clear()
x.plot(s,line,tmpl)
x.png("bad")
```
bad

### PR DESCRIPTION
**Pull request by doutriaux1**
@danlipsa @aashish24 ready for review

### CODE REVIEW

**Comment by doutriaux1** [comment]
@danlipsa this is ready to go in

**Comment by danlipsa** [comment]
@doutriaux1 Do you have an image that goes with the test?

**Comment by doutriaux1** [comment]
@danlipsa it's referenced at beg of the PR: https://github.com/UV-CDAT/uvcdat-testdata/pull/182

File: vcs/template.py [hunk 1]
```diff
@@ -1175,49 +1190,12 @@ def drawTicks(self, slab, gm, x, axis, number,
                                 end < -180.:
                             end = wc[0]
                         xs.append([wc[0], end])
-                        tys.append(l_tmp)
-                        txs.append(wc[0])
-                    tstring.append(loc[l_tmp])
-        # now does the mini ticks
-        mintics = getattr(gm, axis + 'mtics' + number)
-        if mintics != '':
```
Comment (danlipsa) [inline vcs/template.py]: Why is this chunk removed? (if mintics != '')

Comment (doutriaux1) [reply vcs/template.py]: that was the guilty part, it was running the mintics with the same property as the main tics, i now call this thing once for main tics and once for mintics (new input keyword) so we can use the different properties.

**Comment by danlipsa** [comment]
LGTM

**Pull request merged**